### PR TITLE
Enable extension for certain sites

### LINKED ISSUE #38: Enable extension for certain sites

### LINKED ISSUE #56: Add a special layout for when the media is not rated by Shepherd

### PR DESCRIPTION
**Pull request by tselmek**
Close #38 
Close #56

### CODE REVIEW

File: extension/shepherd/popup.js [hunk 1]
```diff
@@ -133,6 +161,33 @@ function populateContent() {
...
+
+/**
+ * Populate the extension's main window
+ */
+function populateContent() {
+  // Fetch media from API based on current page's domain
+  let uri = getCurrentPageUri();
+
+  if (!isKnownMedia(uri)) {
+    currentMedia = fetchMediaByWebsite(uri);
```
Comment (pBouillon) [inline extension/shepherd/popup.js:183]: If the media is not known, fetching it won't work

File: extension/shepherd/popup.js [hunk 2]
```diff
@@ -113,16 +137,20 @@ function getTrimmed(value) {
...
+ * @param {string} uri - Website URI of the media
+ * @return {boolean} - Boolean of whether the media exists in the database
  */
-function populateContent() {
-  // Fetch media from API based on current page's domain
-  let uri = getCurrentPageUri();
-  currentMedia = fetchMediaByWebsite(uri);
-  
+function isKnownMedia(uri) {
+  return false;
```
Comment (pBouillon) [inline extension/shepherd/popup.js:144]: It should be an HTTP call now

**Review by pBouillon — CHANGES_REQUESTED** [review]

File: extension/shepherd/popup.js [hunk 3]
```diff
@@ -13,6 +13,21 @@ const api = axios.create({
...
+ * @const {string} bodyForUnknownMedia - HTML string for the body content of unknown media
+ */
+const bodyForUnknownMedia = `
+  <div class="container">
+    <div class="row text-center h-50 d-flex align-content-center">
+      <p class="pt-4">This website is not yet rated by Shepherd</p>
+    </div>
+
+    <div class="row text-center h-50 d-flex align-content-center">
+      <p class="pt-2">Help the community by suggesting this media on the <a class="shepherd-link" href="${config.websiteUri}" target="_blank">Shepherd website</a></p>
```
Comment (pBouillon) [inline extension/shepherd/popup.js:26]: I would highlight "suggesting this media" instead

File: extension/shepherd/config.js [hunk 4]
```diff
@@ -3,6 +3,7 @@
  */
 export const config = {
     apiUri: "http://localtest.me:8080/api/",
+    websiteUri: "http://localtest.me:4200/",
```
Comment (pBouillon) [inline extension/shepherd/config.js:6]: This should be retrieved from the current URI

Comment (tselmek) [reply extension/shepherd/config.js:6]: This is the Shepherd website's page but I agree the variable's name is not clear

**Review by pBouillon — APPROVED** [review]

**Pull request merged**